README.md

```diff
@@ -1,82 +1,24 @@
 
 ## How to use Trin
 
-Checkout out the [Getting Started](/docs/getting_started.md) guide to quickly get up and running with Trin.
+Check out the [Getting Started](/docs/getting_started.md) guide to quickly get up and running with Trin.
 
-## CLI Options
-```sh
-trin 0.0.1
-carver
-Run an eth portal client
+## Experimental Status
 
-USAGE:
-    trin [OPTIONS]
+Trin is a prototype Portal Network client. This implementation and the Portal Network specificiation will continue to co-evolve.
 
-FLAGS:
-        --enable-metrics    Enable prometheus metrics reporting (requires --metrics-url)
-    -h, --help              Prints help information
-        --internal-ip       (For testing purposes) Use local ip address rather than external via STUN.
-    -V, --version           Prints version information
+In this stage of development, Trin relies on Infura to respond to some requests that it cannot fulfill itself. Additionally, Trin lacks comprehensive data validation.
 
-OPTIONS:
-        --bootnodes <bootnodes>               One or more comma-delimited base64-encoded ENR's or multiaddr strings of
-                                              peers to initially add to the local routing table [default: ]
-        --discovery-port <discovery_port>     The UDP port to listen on. [default: 9000]
-        --external-address <external_addr>    The public IP address and port under which this node is accessible
-        --kb <kb>                             Maximum number of kilobytes of total data to store in the DB
-                                              [default: 100000]
-        --metrics-url <metrics-url>           URL for prometheus server
-        --networks <networks>...              Comma-separated list of which portal subnetworks to activate
-                                              [default: history,state]
-        --pool-size <pool_size>               max size of threadpool [default: 2]
-        --unsafe-private-key <private_key>    Hex encoded 32 byte private key (considered unsafe to pass in pk as cli
-                                              arg, as it's stored in terminal history - keyfile support coming soon)
-        --web3-http-address <web3-http-address>    address to accept json-rpc http connections [default: 127.0.0.1:8545]
-        --web3-ipc-path <web3_ipc_path>       path to json-rpc endpoint over IPC [default: /tmp/trin-jsonrpc.ipc]
-        --web3-transport <web3_transport>     select transport protocol to serve json-rpc endpoint [default: ipc]
-                                              [possible values: http, ipc]
-```
+Do not rely on Trin in a production setting.
 
 ## Want to help?
 
 Want to file a bug, contribute some code, or improve documentation? Excellent! Read up on our
 guidelines for [contributing](/docs/contributing.md),
 then check out issues that are labeled
 [Good First Issue](https://github.com/ethereum/trin/issues?q=is%3Aopen+is%3Aissue+label%3A%22good+first+issue%22).
```

Comment:
A link to our discord channel could also be useful here. Something like:  "Have questions? Want to join the conversation? Come chat with us on discord."